### ISSUE TITLE
Request - Markdown support

### ISSUE THREAD

**Issue by henriiquecampos** [itchio/itch.io#677]
Hello there, first of all thanks for the excellent platform you guys provide you're doing the most awesome thing I can imagine for indie developers, seriously.

Straight to the point, I would really like to edit my posts, devlogs, comments, page et cetera with markdown, is is possible?

**Comment by rm-code** [comment]
This so much ... looks like now we can even post in HTML formats directly. This makes it really annoying to cross-post between different pages and itch.io.

**Comment by bjorn** [comment]
Ah, exactly my frustration every time I write a devlog! Instead of being able to write some Markdown or even HTML in my favorite editor, I'm forced to use the clumsy WYSIWYG online editor. :-(

**Comment by charliesgames** [comment]
It would be handy to push devlogs via butler too.

Cheers
Charlie

**Comment by aure2006** [comment]
WYSIWYG is so cancerous and for low IQ people. So annoying to mess with this crap.

**Comment by Guard13007** [comment]
@charliesgames Sounds awesome, but should be a separate issue.

**Comment by nicross** [comment]
Adding my support to this feature request. The WYSIWYG toolbar is inaccessible, meaning that NVDA users can't even switch to HTML mode to perform formatting like headings and lists. If all multi-line text editors on itch.io respected the "Prefer Markdown input where available" profile setting, then these users could access more formatting options.

**Comment by ndarilek** [comment]
Universal Markdown support would be amazing, but if the issue is lack of resources, I'd personally like to see the various game edit screens added next. I'm a blind game developer wanting to publish on Itch, and we're hosting an audio game jam next month. As of now, I'm having a hard time inputting rich text into my game description.

Thanks.

**Comment by leafo** [comment]
@ndarilek Do you find the accessibility of the rich text editor no good, or you just prefer to work with plain text?

Thanks

**Comment by ndarilek** [comment]
The latter. I have my game's description in a README.md that Pandoc 
converts to HTML for the manual. I then paste either the Markdown or the 
HTML into various places, whichever makes the most sense. I'd rather not 
write out a new description for each platform if I can help it.


Though, accessibility-wise, I don't know how I'm supposed to enter HTML 
into the editor. Do I paste in HTML, then click the HTML button? Do I 
click HTML, then paste it in? It isn't entirely clear, but what *would* 
be clear is being able to just use Markdown. :)


Thanks.

**Comment by ndarilek** [comment]
FYI, I've successfully inserted HTML into the editor by clicking the 
HTML button then performing my paste. I'm using Orca, so can't comment 
on other screen reader accessibility.


Would still be nice to see Markdown support in more places, but for the 
time being, I'm able to move ahead with publishing my game.

**Comment by aure2006** [comment]
My main problem is the current editor automatically making changes instead of only when I specifically say.
Example:
Paste a URL, then press space and start typing. Any new text will become part of the URL.

I'd prefer just working with plain text.

**Comment by gingerbeardman** [comment]
Another plain text fan here, currently struggling with the WYSIWYG editor waist editing my project description: it keeps inserting paragraphs in the middle of lists, and other weirdness. 

I'd prefer to type markdown.

**Comment by octoshrimpy** [comment]
another one in favor of markdown in the project description. (also @gingerbeardman I keep running into you everywhere!)

**Comment by fguillen** [comment]
Yes, yes, please markdown support for the project description. It is insanely annoying to have to reformat each time I want to update my README.md (using markdown) and then in itch.io using its personal one :/

**Comment by Tarroook** [comment]
Please markdown support would be so handy

**Comment by ghost** [comment]
Markdown support for Project Descriptions would go a long way!

**Comment by RokeJulianLockhart** [comment]
> https://github.com/itchio/itch.io/issues/677#issue-242124501

Consider adhering to CommonMark, albeit with myriad necessary extensions (like table support) and basic HTML integration, akin to GLFM and GHFM.

Additionally, for the sake of those subscribed, don't comment when an upvote would suffice.

**Comment by gingerbeardman** [comment]
Yikes

**Comment by HorrellTech** [comment]
Any luck? Cant be that difficult to add. Just use Github Copilot to assist if its too difficult. @devs

**Comment by CypherPoet** [comment]
Commenting with my own support to help get some visibility on this 🙏
